templates/baseline/baseline.jinja2

```diff
@@ -1,3 +1,33 @@
+- test:
+    timeout:
+      minutes: 10
+    definitions:
+    - repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: baseline
+          description: "baseline test plan"
+          os:
+            - debian
+          scope:
+            - functional
+          environment:
+            - lava-test-shell
+        run:
+          steps:
+            - dmesg --level=warn --notime -x -k > dmesg.warn
+            - >
+                for level in err crit alert emerg; do
+                  dmesg --level=$level --notime -x -k > dmesg.$level
+                  test -s dmesg.$level && res=fail || res=pass
+                  count=$(cat dmesg.$level | wc -l)
+                  lava-test-case $level --result $res --measurement $count
```

Comment:
We should probably have a `--unit` here.  The unit will be used by the frontend (and email reports) .  The unit could just be "messages"

Otherwise, LGTM

Reply:
Sure, I'll add that.

Reply:
I opted for "lines" as it seemed more accurate.  Some messages may span across several lines such as a kernel oops, but we're not parsing those to group all the lines as a single message (yet).